cocos/renderer/core/gfx/GFXCommandBuffer.h

```diff
@@ -34,16 +34,27 @@ class CC_DLL CommandBuffer : public GFXObject {
     virtual void updateBuffer(Buffer *buff, const void *data, uint size, uint offset) = 0;
     virtual void copyBuffersToTexture(const uint8_t *const *buffers, Texture *texture, const BufferTextureCopy *regions, uint count) = 0;
     virtual void execute(const CommandBuffer *const *cmdBuffs, uint32_t count) = 0;
+    
+    CC_INLINE void bindDescriptorSetJS(uint set, DescriptorSet *descriptorSet, uint dynamicOffsetCount, const std::vector<uint> &dynamicOffsets) {
```

Comment:
i think `bindDescriptorSetForJS` is more meaningful.

Reply:
Can we just skip the specific overload? Or if that takes time to implement, at least remove the `dynamicOffsetCount` parameter since it comes with the vector already.